indico/modules/events/registration/fields/simple.py

```diff
@@ -389,3 +390,24 @@ def _indico_email(value):
                 raise ValidationError(_('Invalid email address'))
 
         return _indico_email
+
+
+class PictureField(FileField):
```

Comment:
I think you want `mm_field_class = MMFileField` (and `from indico.util.marshmallow import FileField as MMFieldField`) - that way you do not need to manually query the file below but get it nicely as the field's `value`

Reply:
I am getting a disparity here because the `PictureField` inherits from the `FileField` which uses `mm_field_class = UUIDString` so If I change the `mm_field_class` here I would also then need to define my own `process_form_data` function for it to not break something.

Reply:
@ThiefMaster, please do you have any other suggestion for this?

Reply:
Nevermind in that case - I didn't realize we're doing the exact same thing for the `FileField`.... something to refactor later I guess

Reply:
okay